app.js

```diff
@@ -93,3 +94,7 @@ playButton.addEventListener('click', () => {
     }
 
 }); 
+
+resetButton.addEventListener('click', () => {
+    document.location.reload();
```

Comment:
this is reloading the page. You will want to do something like :
total++,
wins = 0,
losses = 0, 
etc.  and reset the text content. 
You want to reset the state not reload the page.